kytos/core/buffers.py

```diff
@@ -40,7 +44,12 @@ def put(self, event):
             self._reject_new_events = True
 
     def get(self):
-        """Remove and return a event from top of queue."""
+        """Remove and return a event from top of queue.
+
+        Returns:
+            :class:`~kytos.core.buffers.KytosEventBuffer`:
```

Comment:
This is not the correct class.
`kytos.core.buffers.KytosEventBuffer` != `kytos.core.events.KytosEvent`